### ISSUE TITLE
A11y_ConnectedFieldService-IoTAlerts_SendCommand_ScreenReader:NVDA announces the selected list twice under device list box.

### ISSUE THREAD

**Issue by V-sheman** [nvaccess/nvda#10369]
"Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier."
​GitHubTags: #A11yMAS; #A11yTCS; #; #MAS1.3.1; #A11y_ConnectedFieldService; #Win32; #DesktopWeb
**Environment Details:​**
Application Name: Connected Field Service- Solution Deployment App
User Name – OCETestTenant@OCERegion.onmicrosoft.com
Password – Mic**S***t
Operating System: Win10​
Edge:44.17763.1.0
NVDA version - 2019.2

**Repro Steps:​**
1. Launch the URL https://monti1tiecrmlivesg619org0.crm2.crmlivetie.com 
2.Navigate to site map and Select IoT Alerts.
3.Navigate to any record from the grid and click on it.
4.Navigate to "SendCommand" menu item and hit enter.​
5. Navigate to " SendCommand " Screen.

6.Start NVDA observe the announcement.

**Actual:​**
NVDA announcing the list item twice for example "Device Device 1 of 1" 

**Expected:​**
NVDA should announce Device Device  twice it should announce once only for ex Device 1 of 1.

**​User Impact:​**
Screen reader users will be confuse by the dual announcement and will misunderstood the content.  
**Recommendations:​**
Please refer to below URL which is repository for accessibility bug fixes.​
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx​

​**MAS Reference:​**
MAS1.3.1:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**This is external to** https://dynamicscrm.visualstudio.com/DefaultCollection/OneCRM/_workitems/edit/1562203/?triage=true

**Comment by dingpengyu** [comment]
hi @V-sheman 
Please follow ISSUE_TEMPLATE to fill in your question
thanks
You have arrived at a default issue template (bug report).
It is preferable to [choose the type of issue](https://github.com/nvaccess/nvda/issues/new/choose) to submit.
Hint: Swap to the preview tab to be able to click the links.
Direct links to the templates:
- [Bug report](https://github.com/nvaccess/nvda/issues/new?template=bug_report.md)
- [Feature request](https://github.com/nvaccess/nvda/issues/new?template=feature_request.md)



### Steps to reproduce:

### Actual behavior:

### Expected behavior:

### System configuration
#### NVDA installed/portable/running from source:

#### NVDA version:

#### Windows version:

#### Name and version of other software in use when reproducing the issue:

#### Other information about your system:

### Other questions
#### Does the issue still occur after restarting your PC?

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

**Comment by michaelDCurran** [comment]
Closing on request from reporter privately by email.
